### PR TITLE
2883 bind to model

### PR DESCRIPTION
**Pull request by easherma**
# Description

Forked off wagtail-modeltranslation (https://github.com/cityofaustin/wagtail-modeltranslation) to push a change to make it wagtail 2.7 compliant 

Fixes https://github.com/cityofaustin/techstack/issues/2883

### CODE REVIEW

**Comment by Niicck** [comment]
Again, the auto-linter you're using makes it really hard to see what changes you made here: https://github.com/cityofaustin/wagtail-modeltranslation/commit/dd2f05a9dd0e9d3adad557082610e4b898873bde

Was it necessary for us to do a fork off wagtail-modeltranslation? If this was an issue for other people, couldn't this be a PR into wagtail-modeltranslation so we can stay on the same level as the rest of the wagtail community?

**Comment by easherma** [comment]
@Niicck 
The line changed is right here: https://github.com/cityofaustin/wagtail-modeltranslation/blob/dd2f05a9dd0e9d3adad557082610e4b898873bde/wagtail_modeltranslation/patch_wagtailadmin.py#L152

I did open a PR on the original repo: 
https://github.com/infoportugal/wagtail-modeltranslation/pull/257

However, they have some pretty old PRs sitting around that haven't gotten reviewed at all(its not a ore wagtail project), so I'm not sure how active the original author is and didn't want to wait on them for a simple change.

**Comment by Niicck** [comment]
Cool beans. At my initial pass at this issue, I didn't catch that it was one of our imported modules that was throwing the error. Good catch.

Would it be faster/easier/better to deploy this and future solutions if git was installed on the Joplin Docker container? That's an easy thing we can add.

Can you add documentation about the forked module in Joplin?

**Comment by easherma** [comment]
Re: git in docker, I decided last time and still agree that this current flow is fine as long as we aren’t doing a ton of development work on these modules. If that changes in the future, it might be worth it. As is, we rarely have to update them and I know how to do it and it dosen’t take long.

Re: documentation, I could add that somewhere. They are self documenting to the extent that requirements.txt notes where they come from.

**Comment by Niicck** [comment]
Correct, the requirements.txt does say that we're using a module called coa-wagtail-modeltranslation. But it's a bigger deal than just using another module. We now have another mission-critical repo maintained by us that's vital to the deployment of the project. This is an unusual, non obvious thing that could easily get lost to tribal knowledge within a year. It definitively needs an explanation in the README. Maybe a subsection for "related repos" that I could also add my publisher documentation to?

File: README.md [hunk 1]
```diff
@@ -340,6 +341,28 @@ We're using webpack to bundle syles and scripts, and webpack_loader to include t
...
+https://github.com/cityofaustin/wagtail-modeltranslation
+
+We use wagtail-modeltranslation to handle translated fields, this fork corrects a single line of code to make it compatible with current and future versions of wagtail.
+
+### Publisher
+
+https://github.com/cityofaustin/publisher
+
+Microservice that handles publishing versions of Janis based on branches of Joplin.
+
```
Comment (robbobfrh84) [inline README.md]: 👍docs!

**Pull request merged**